src/test/scala/org/kframework/minikore/TextToMiniTest.scala

```diff
@@ -61,347 +61,443 @@ class TextToMiniTest {
     val s =
-      """
+      strip("""
         |[]
         |module A
         |  syntax X
         |endmodule
-        |""".stripMargin
+        |""")
     try {
       parseFromString(s)
       assert(false)
     } catch {
       case ParseError(msg) =>
         assertEquals(
-          """
-            |ERROR: Line 5: Column 1: Expected '[' or ':', but 'e'
+          strip("""
+            |ERROR: Line 4: Column 1: Expected '[' or ':', but 'e'
             |endmodule
             |^
-            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
+            |"""),
           msg)
     }
   }
 
   @Test def parseTestFail13(): Unit = {
     val s =
-      """
+      strip("""
         |[]
         |module A
         |  dummy
         |endmodule
-        |""".stripMargin
+        |""")
     try {
       parseFromString(s)
       assert(false)
     } catch {
       case ParseError(msg) =>
         assertEquals(
-          """
-            |ERROR: Line 4: Column 3: Expected import, syntax, rule, axiom, or endmodule, but 'd'
+          strip("""
+            |ERROR: Line 3: Column 3: Expected import, syntax, rule, axiom, or endmodule, but 'd'
             |  dummy
             |  ^
-            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
+            |"""),
           msg)
     }
   }
 
   @Test def parseTestFail14(): Unit = {
     val s =
-      """
+      strip("""
         |[]
         |module _A
         |endmodule
-        |""".stripMargin
+        |""")
     try {
       parseFromString(s)
       assert(false)
     } catch {
       case ParseError(msg) =>
         assertEquals(
-          """
-            |ERROR: Line 3: Column 8: Expected <ModuleName>, but '_'
+          strip("""
+            |ERROR: Line 2: Column 8: Expected <ModuleName>, but '_'
             |module _A
             |       ^
-            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
+            |"""),
           msg)
     }
   }
 
   @Test def parseTestFail15(): Unit = {
     val s =
-      """
+      strip("""
         |[]
         |module A
         |  syntax X ::= x(Y Z) []
         |endmodule
-        |""".stripMargin
+        |""")
     try {
       parseFromString(s)
       assert(false)
     } catch {
       case ParseError(msg) =>
         assertEquals(
-          """
-            |ERROR: Line 4: Column 20: Expected ')' or ',', but 'Z'
+          strip("""
+            |ERROR: Line 3: Column 20: Expected ')' or ',', but 'Z'
             |  syntax X ::= x(Y Z) []
             |                   ^
-            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
+            |"""),
+          msg)
+    }
+  }
+
+  @Test def parseTestFailWhiteSpacesPrefix(): Unit = {
+    val s =
+      """
+
+
+
+        |[]
+        |module A
+        |endmodule [}
+        |""".stripMargin
```

Comment:
This is not a valid multiline string in Scala. You missed the bar `|` in the beginning. Is that your intention? Also, you missed applying `strip`; is this also your intention?